pkg/local_object_storage/metabase/graveyard.go

```diff
@@ -9,70 +9,245 @@ import (
 	"go.etcd.io/bbolt"
 )
 
-// Grave represents descriptor of DB's graveyard item.
-type Grave struct {
-	gcMark bool
-
+// GarbageObject represents descriptor of the
+// object that has been marked with GC.
+type GarbageObject struct {
 	addr *addressSDK.Address
 }
 
-// WithGCMark returns true if grave marked for GC to be removed.
-func (g *Grave) WithGCMark() bool {
-	return g.gcMark
+// Address returns buried object address.
```

Comment:
in fact, took that term from the tests' comments (and started to name all GCed/ TSed objects as "buried"/"deleted" and the file is still named as "graveyard" cause all those objects gonna "die" anyway). do not have my own terminology but agree that it would be better to name objects with GC as "garbage" and objects with TS as "tombstoned" everywhere in the repo

changed